kimchi/src/circuits/polynomials/keccak/mod.rs

```diff
@@ -98,6 +103,82 @@ pub(crate) fn expand_word<F: PrimeField, T: ExprOps<F>>(word: u64) -> Vec<T> {
         .collect::<Vec<T>>()
 }
 
+/// Pads the message with the 10*1 rule until reaching a length that is a multiple of the rate
+pub(crate) fn pad(message: &[u8]) -> Vec<u8> {
```

Comment:
![where-is-your-5c9ae4](https://github.com/o1-labs/proof-systems/assets/6018454/53a01866-ba72-4570-9c95-b70da7782761)

Reply:
Would have been nice to have some tests with corner cases.
What about messages of 1088 bits? The documentation does not mention it. Reading the code makes me feel it adds a full block. But tests can confirm this.